### ISSUE TITLE
XGBOOST and  CV

### ISSUE THREAD

**Issue by jejen3180** [IvanLetteri/MTA-KDD-19#3]
I tested your dataset with xgboost and cross validation, the result is accuracy of 100% and Actually, the average accuracy of all the experiments was k-fold 5 score.mean 99,31%

**Comment by IvanLetteri** [comment]
Well done, good work jejen3180. We known about excellent results about xgboost and I say you that also different decision trees perform well, better than neural networks (see https://ieeexplore.ieee.org/document/8802421).
 
We work on deep neural network doing research about NN-learning steps so that was the reason which induce to use NN.

**Comment by jejen3180** [comment]
if you are willing, I would like to use your dataset for my research ( I use XGBoost), can you give me some input, please..?
